apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/index.tsx

```diff
@@ -1,23 +1,46 @@
-import { Flex, InfiniteLoader, Text, Button } from "@ledgerhq/native-ui";
-import React, { useState } from "react";
+import { Flex, InfiniteLoader } from "@ledgerhq/native-ui";
+import React, { useState, useMemo } from "react";
 import { useLargeMover } from "./hooks/useLargeMover";
 import getWindowDimensions from "~/logic/getWindowDimensions";
 import { Card } from "./components/Card";
 import { LandingPagesNavigatorParamList } from "~/components/RootNavigator/types/LandingPagesNavigator";
-import { ScreenName } from "~/const";
+import { NavigatorName, ScreenName } from "~/const";
 import { StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
 import { StickyHeader } from "./components/StickyHeader";
 import { SafeAreaView } from "react-native";
 import { useTheme } from "styled-components/native";
+import { SwiperComponent } from "LLM/components/Swiper/components/Swiper";
+import {
+  CurrencyData,
+  KeysPriceChange,
+  MarketCoinDataChart,
+} from "@ledgerhq/live-common/market/utils/types";
+import { NavigationProp, useNavigation } from "@react-navigation/native";
+import { WalletTabNavigatorStackParamList } from "~/components/RootNavigator/types/WalletTabNavigator";
 
 type LargeMoverLandingPageProps = StackNavigatorProps<
   LandingPagesNavigatorParamList,
   ScreenName.LargeMoverLandingPage
 >;
 
+type Card = {
+  data?: CurrencyData | undefined;
+  chartData?: MarketCoinDataChart | undefined;
+  range: KeysPriceChange;
+  setRange: (range: KeysPriceChange) => void;
+  isLoading: boolean;
+  isError: boolean;
+};
+
+const LoadingIndicator = ({ height }: { height: number }) => (
+  <Flex height={height} width="100%" justifyContent="center" alignItems="center">
+    <InfiniteLoader color="primary.c50" size={38} />
+  </Flex>
+);
+
```

Comment:
you can create a LoadingComponent in `components/`

Also I don't think `height ` is needed, just use 100% as  the component will be a children